getPriceFromAmazon.py

```diff
@@ -1,10 +1,20 @@
 import requests
 import bs4
 
-def getPrice(ProductID):
+def getPrice(URL):
+
+    country_prefix = ""
+
+    if "amazon.de" in URL:
+        country_prefix = "de."
+        URL = URL.replace("/dp/", "/product/")
+
+    URL = URL.replace("?", "/")
+    ProductID = URL.split("/product/", 1)[1].split("/")[0]
+    print(ProductID)
```

Comment:
Is there any purpose behind printing the ProductI? from my point of view this should be eliminated.

Reply:
Oh, That was only for testing purposes. I will remove it.